pkg/agent/multicast/mcast_controller_test.go

```diff
@@ -42,6 +42,11 @@ import (
 	"antrea.io/antrea/pkg/util/channel"
 )
 
+var (
+	queryInterval     = 125 * time.Second
+	mcastGroupTimeout = queryInterval * 3
```

Comment:
When shall we use `mcastGroupTimeout`? It is auto-generated when constructing Controller?

BTW, maybe use a small value in the unit test? I remember there are some existing cases overwrite the values. Could you modify them?

Reply:
Yes, it is in unit test `TestClearStaleGroups`, changed `queryInterval` to `5s` and `mcastGroupTimeout` to `15s`

Reply:
what about mcastGroupTimeout?

Reply:
I mean both queryInterval and mcastGroupTimeout are in TestClearStaleGroups.

Reply:
removed